webapp/security/models.py

```diff
@@ -135,39 +119,40 @@ def support_tag(self):
         return ""
 
 
-class Bug(Base):
-    __tablename__ = "bug"
+class Status(Base):
+    __tablename__ = "status"
 
-    id = Column(Integer, primary_key=True)
-    uri = Column(String)
-
-
-class PackageStatus(Base):
-    __tablename__ = "package_status"
-
-    id = Column(Integer, primary_key=True)
-    status = Column(String)
-    status_description = Column(String)
-
-    package_id = Column(Integer, ForeignKey("package.id"))
-    package = relationship("Package", back_populates="package_statuses")
+    cve_id = Column(String, ForeignKey("cve.id"), primary_key=True)
+    package_name = Column(String, ForeignKey("package.name"), primary_key=True)
```

Comment:
So this is something that is failing but seems a bit contradictory. Status needs a `package_name` but when I give it status.package_name in the bulk insert data, it says `Unknown field` I suppose it's because it's not define in the schema? It's being returned as an error.message after trying the schema_load

Reply:
I don't understand... this works for me, this is probably another issue with me not having pushed the latest script, which I've now done. Please try again with the changes to the script.